app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -358,6 +358,7 @@ interface CustomizableOptionInterface @typeResolver(class: "Magento\\CatalogGrap
     title: String @doc(description: "The display name for this option")
     required: Boolean @doc(description: "Indicates whether the option is required")
     sort_order: Int @doc(description: "The order in which the option is displayed")
+    option_id: Int @doc(description: "Option ID")
```

Comment:
@VoronoyAlexandr  Why does `option_id` need to be exposed? Please provide scenario, which requires this.

Reply:
Good day @paliarush.
Scenario:
If we used mutation to add product in cart with options like in this code
```
mutation {
  addSimpleProductsToCart(
    input: {
      cart_id: "CDNuOMoQBV3CElFSUbgbAswlVcoaeSId", 
      cartItems: [
        {
          data: {
            sku: "product_dynamic_1", 
            qty: 1
          }, 
          customizable_options: [
            {id: 5, value: "4"},
            {id: 6, value: "5"}
          ]
        }
      ]
    }
  ) 
.......
```
we need know what is id of options. At this moment schema return only 'title', 'required', 'sort_order'. This option_id in schema, response look like:

```
{
  "data": {
    "products": {
      "items": [
        {
          "id": 1,
          "attribute_set_id": 10,
          "created_at": "2018-11-11 14:00:34",
          "name": "Simple Product 1",
          "sku": "product_dynamic_1",
          "type_id": "simple",
          "updated_at": "2018-11-13 09:25:55",
          "options": [
            {
              "title": "Test option 1",
              "required": true,
              "sort_order": 1,
              "option_id": 1
            },
            {
              "title": "Test option 2",
              "required": true,
              "sort_order": 2,
              "option_id": 3
            }
          ]
        }
      ]
    }
  }
}
```